controllers/admin/AdminImagesController.php

```diff
@@ -32,18 +32,198 @@ class AdminImagesControllerCore extends AdminController
     protected $start_time = 0;
     protected $max_execution_time = 7200;
     protected $display_move;
+    protected $generateAdditionalAvif;
 
     public function __construct()
     {
         $this->bootstrap = true;
         $this->table = 'image_type';
         $this->className = 'ImageType';
-        $this->lang = false;
 
+        parent::__construct();
+    }
+
+    public function init()
+    {
         $this->addRowAction('edit');
         $this->addRowAction('delete');
 
-        parent::__construct();
+        parent::init();
+
+        $this->generateAdditionalAvif = $this->get('prestashop.core.configuration.avif_extension_checker')->isAvailable();
+
+        $imageFormatsDisabled = [];
+
+        if (false === $this->generateAdditionalAvif) {
+            $imageFormatsDisabled['avif'] = true;
+        }
+
+        $fields = [
+            'PS_IMAGE_FORMAT' => [
+                'title' => $this->trans('Image format', [], 'Admin.Design.Feature'),
+                'show' => true,
+                'required' => true,
+                'skipIsCleanHtml' => true,
+                'type' => 'checkbox',
+                'multiple' => true,
+                'choices' => [
+                    'jpeg' => $this->trans('JPEG', [], 'Admin.Design.Feature'),
+                    'png' => $this->trans('PNG', [], 'Admin.Design.Feature'),
+                    'webp' => $this->trans('WebP', [], 'Admin.Design.Feature'),
+                    'avif' => $this->trans('AVIF', [], 'Admin.Design.Feature'),
+                ],
+                'value_multiple' => [
+                    'jpeg' => false,
+                    'png' => false,
+                    'webp' => false,
+                    'avif' => false,
+                ],
+                'disabled' => $imageFormatsDisabled,
+            ],
+            'PS_IMAGE_FORMAT_FALLBACK' => [
+                'title' => $this->trans('Fallback', [], 'Admin.Design.Feature'),
+                'validation' => 'isUnsignedId',
+                'required' => false,
+                'cast' => 'intval',
+                'type' => 'select',
+                'list' => [
+                    [
+                        'id' => '0',
+                        'name' => $this->trans('JPEG', [], 'Admin.Design.Feature'),
```

Comment:
I'm not sure if those should be translatable. There's still some time. Maybe one helper class should provide these options, so you can have one file where you change available image extensions.